reference/public-ip-addresses.md

```diff
@@ -18,3 +20,23 @@ North American region (us.platform.sh):
 These addresses are stable, but not guaranteed to never change. Prior to
 any future change, all affected customers will receive ample warning.
 
+## Inbound IP addresses
+
+Platform.sh regions accept connections on a limited number of IP
+addresses. You may need to whitelist them on your corporate firewall
+to enable SSH connections.
```

Comment:
Ahhh...  That's not what the text here implies. The intro text should therefore be rewritten.  (The issue isn't on our end, it's on your end if your firewall limits outgoing ssh connections.)